### DISCUSSION TITLE
macOS VM Install

### DISCUSSION THREAD

**Discussion by MCES10** [utmapp/UTM#5446] [UTM Help (macOS)]
**Configuration**  
* UTM Version: 
* macOS Version: 
* Mac Chip (M1Pro): 

I have a issue when moving an .utm file to a external drive which is formatted to exFAT isn't allowing me to install macOS and instead crashes during file transfer and gives a corrupted utm file or macOS doesn't install I let to install for multiple hours but it wouldn't install and was stuck on 0%.

**Comment by osy** [comment]
Your drive may be too slow. This is an macOS issue unfortunately. Had similar issues trying to install a hackintosh on a slow drive.

**Reply by MCES10** [reply]
It used to work on old utm versions and worked fine when I used ventura beta last year thats why I am confused.
